pkg/util/common.go

```diff
@@ -122,3 +126,47 @@ func FakeGOPATH(srcDir string) (string, error) {
 
 	return gitRoot, nil
 }
+
+// UntarAndRead opens a tarball and reads contents of a file inside.
+func UntarAndRead(tarPath, filePath string) (string, error) {
```

Comment:
Do we wanna write some tests for this function and `MoreRecent`?

Reply:
Definitely! In my check list above

Reply:
nits:
- Do we maybe want to return the `io.Reader` instead of a string? That way it
  - would be more ideomatic
  - a caller could wrap a LimitedReader around it or do whatever
- Is a name like `ReadFileFromGzippedTar(...)` a "truer" name. Would it be a better name? Not really ... 🤔#namingThings

Reply:
Agree to both of these points :) This structure was mostly a result of trying to mimic the exact behavior of https://github.com/kubernetes/release/blob/471139de81b6e0477d6c5438e7c143b2b26c07c7/push-build.sh#L131